pandas/core/ops.py

```diff
@@ -168,6 +169,9 @@ def add_special_arithmetic_methods(cls, arith_method=None,
         f(op, name, str_rep, default_axis=None, fill_zeros=None, **eval_kwargs)
     comp_method : function, optional,
         factory for rich comparison - signature: f(op, name, str_rep)
+    divmod_method : function, (optional)
```

Comment:
I wasn't sure if we wanted the API for dataframes to be a return of a tuple with the div result and the mod result but for consistency it could work. Also, there is no reflected or inplace divmod. I construct this function from `_arith_method_SERIES` right now but could move this into `_create_methods`